DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -326,6 +326,7 @@ final class AddressBarTextField: NSTextField {
 
     private func navigate(suggestion: Suggestion?) {
         let ntpExperiment = NewTabPageSearchBoxExperiment()
+        let ntpExperimentCohort: NewTabPageSearchBoxExperiment.Cohort? = ntpExperiment.isActive ? ntpExperiment.cohort : nil
```

Comment:
For autocomplete pixels we don't want to pass the Search Box (or onboarding) cohort when experiment is inactive.